static/app/components/replays/utils.tsx

```diff
@@ -24,14 +24,14 @@ const HOUR = 60 * MINUTE;
  * @returns Unix timestamp of the adjusted timestamp, in milliseconds
  */
 export function relativeTimeInMs(timestamp: moment.MomentInput, diffMs: number): number {
-  return moment(timestamp).diff(moment.unix(diffMs)).valueOf();
```

Comment:
I think this was added due to some timezone issues? -- I don't think we should be using moment at all here, but that's a diff issue.